api-errors.json

```
@@ -199,6 +199,18 @@
         "short_message": "Malformed Header",
         "long_message": "One or more headers are malformed"
     },
+    "40053": {
+        "short_message": "Unexpected Fields",
```

If the error is forms-specific, I think it should also be in the `short_message`. Same for others. You can also re-use more generic error codes and overwrite the `long_message` in your application, instead of creating very specific errors for forms.

```suggestion
        "short_message": "Unexpected Forms Fields",
```